.idea/artifacts/spotify_web_api_kotlin_js_1_0_SNAPSHOT.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ArtifactManager">
+  <artifact type="jar" name="spotify-web-api-kotlin-js-1.0-SNAPSHOT">
+    <output-path>$PROJECT_DIR$/build/libs</output-path>
+    <root id="archive" name="spotify-web-api-kotlin-js-1.0-SNAPSHOT.jar">
+      <element id="module-output" name="spotify-web-api-kotlin.spotify-web-api-kotlin.jsMain" />
+    </root>
+  </artifact>
+</component>
```

Comment:
You added this file even though it is ignored in the `.gitignore`. Do you want to have it?